libs/application/templates/healthcare-license-certificate/src/lib/HealthcareLicenseCertificateTemplate.ts

```diff
@@ -0,0 +1,147 @@
+import {
+  ApplicationConfigurations,
+  ApplicationTemplate,
+  ApplicationTypes,
+  ApplicationContext,
+  ApplicationRole,
+  ApplicationStateSchema,
+  Application,
+  DefaultEvents,
+  defineTemplateApi,
+  InstitutionNationalIds,
+  NationalRegistryUserApi,
+} from '@island.is/application/types'
+import {
+  EphemeralStateLifeCycle,
+  coreHistoryMessages,
+  corePendingActionMessages,
+  pruneAfterDays,
+} from '@island.is/application/core'
+import { Events, States, Roles } from './constants'
+import { application as applicationMessage } from './messages'
+import { Features } from '@island.is/feature-flags'
+import { ApiActions } from '../shared'
+import { HealthcareLicenseCertificateSchema } from './dataSchema'
+import {
+  UserProfileApi,
+  EmbaettiLandlaeknisPaymentCatalogApi,
+  HealtcareLicenesApi,
+} from '../dataProviders'
+import { buildPaymentState } from '@island.is/application/utils'
+import { getChargeItemCodes } from '../utils'
+
+const template: ApplicationTemplate<
+  ApplicationContext,
+  ApplicationStateSchema<Events>,
+  Events
+> = {
+  type: ApplicationTypes.HEALTHCARE_LICENSE_CERTIFICATE,
+  name: applicationMessage.name,
+  institution: applicationMessage.institutionName,
+  translationNamespaces: [
+    ApplicationConfigurations.HealthcareLicenseCertificate.translation,
+  ],
+  dataSchema: HealthcareLicenseCertificateSchema,
+  featureFlag: Features.healthcareLicenseCertificate,
+  stateMachineConfig: {
+    initial: States.DRAFT,
```

Comment:
The applications should have a seperate prerequisites state that does not list the application and has a short lifecycle.